modules/ROOT/pages/to-download-and-install-studio-lx.adoc

```diff
@@ -3,27 +3,39 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To support some of Studio Themes in Linux, it is recommended to have GTK version 2 installed.
+== Prerequisites
 
-* Before downloading and launching Anypoint Studio, make sure you have the Java SE JDK 8 installed.
+* Ensure that you have the Java SE JDK 8 installed and configured.
+* To support some of Studio Themes in Linux, it is recommended to have GTK version 2 installed.
+* Download the Anypoint Studio installation file from the https://www.mulesoft.com/lp/dl/studio[Anypoint Studio Download Site].
+
+[NOTE]
+Anypoint Studio 7.x only supports Mule 4.x projects since the structure of the project, export format, XML and scripting language are different.
```

Comment:
If you keep this as a Note, suggest the following tweaks:

"Anypoint Studio 7.x supports only Mule 4.x projects, because the structure of the project, export format, XML, and scripting language are different."

However, I am not sure what this means. Are we saying that because Mule 4.x is different (from what?), you can use only Studio 7.x with it?
Or are we saying that because Studio 7.x is different (from what?), you can use it only with Mule 4.x?
Please reword to clarify whether 7.x or 4.x are different from other versions, and therefore work only with ....

Reply:
We had too many users trying to use Studio 7 with Mule 3 projects, which are not compatible.  The reason for this note was to make sure they pay attention to this. 
I like your suggestion of making this a prerequesite, but at the same time, it's not a prerequesite for installing Studio, but a note for users to know that if they install Studio 7, they can only work using Mule 4.x projects.